app/Livewire/Analytics.php

```
@@ -0,0 +1,519 @@
+                    }
+                }
+
+                $attribution ??= $client->attribution();
+
+                // Per-bucket status series; summed by bucket across servers. Isolated so a
+                // series hiccup (or an older Sentinel lacking the endpoint) never discards a
+                // server's other data — an empty result simply flips the chart to the donut.
+                try {
+                    foreach ($client->series($appKey, $this->range) as $bucket) {
+                        $data = $bucket->toArray();
+                        $ts = (int) ($data['bucket'] ?? 0);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ts. Configured minimum length is 3. (undefined)

(ShortVariable)